packages/eslint-plugin-query/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@tanstack/eslint-plugin-query",
+  "version": "1.0.0",
```

Comment:
versions have to be in line with the rest of the monorepo I guess, so let's put this to v4.10.3 please